packages/ui5-application-writer/src/index.ts

```diff
@@ -28,7 +28,7 @@ async function generate(basePath: string, ui5App: Ui5App, fs?: Editor): Promise<
     // Add optional features
     if (ui5App.appOptions) {
         Object.entries(ui5App.appOptions).forEach(([key, value]) => {
-            if (value === true) {
+            if (value !== false) {
```

Comment:
```suggestion
            if (value) {
```
Will a check for 'truthy' values suffice here?

Reply:
The condition is testing for cases where the value is of type boolean and is false explicitly.

Reply:
Is this the explanatory comment?
```
// Add optional features
```

So do we treat an option set to `undefined` or `''` as true?